### ISSUE TITLE
weather prediction

### ISSUE THREAD

**Issue by sheikfaaruk** [AldoStalin/SonaITHackathon#4]
TEAM NAME - Saitama-サイタマ
 PROJECT NAME-Weather prediction
## ℹ️ Project information
          
- **Project Name**:Weather prediction
- **Short Project Description**: Using iot devices to collect the dataset and using the Thingspeak web server
- **Team Name**:Saitama-サイタマ
- **Team Members**:M.Sheik Faaruk (https://github.com/sheikfaaruk) , D.Vignesh (https://github.com/vigneshdhilip02), R.Vasanth.
- **Demo Link**: www.youtube.com/watch?v=dizUklbvoxA
- **Repository Link**:https://github.com/sheikfaaruk/Weather-prediction
- **Labels**: 'IOT','ML'.
## 🔥 Pitch
     IOT Live Weather Station Monitoring Using NodemCU ESP8266 This post is all about IoT based Live Weather Station Monitoring Using NodemCU ESP8266. We will interface DHT11 Humidity & Temperature Sensor, BMP180 Barometric Pressure Sensor, and FC37 Rain Sensor with NodeMCU ESP8266-12E Wifi Module. We will measure humidity, temperature, pressure, and rainfall and upload the data to a web server.Using IOT modules to collect the data save as .csv file format in Thingspeak.com .And using that dataset to find the prediction the weather in it . Prediction is performed using neural nets with backpropagation. Library: Keras.
**Highlights**: We use Both IoT and Machine learning in it 
## ✅ Checklist

- [x] I have followed the issue title format.
- [x] I have applied the correct labels.
- [x] I have mentioned the correct labels.
- [x] I have provided all the information correctly.
